### PR TITLE
Readmes rach java9

### PR DESCRIPTION
**Pull request by Rachael-E**
Second to last of my batches for these reviews, wooo. @hud10837 please could you take a first review?

- VectorTiledLayerURL
- ViewContentBeneathTerrainSurface
- ViewPointCloudDataOffline
- ViewshedCamera
- ViewshedGeoElement -> note for this one, I kept the capitalization of the name since the name includes a class name. "geoElement" looks wrong as does "geo element" or "geoelement".

### CODE REVIEW

**Comment by Rachael-E** [comment]
thanks @hud10837! Changes made. @sorenoid could you do a second review please?

File: java/viewshed-camera/README.md [hunk 1]
```diff
@@ -1,24 +1,37 @@
...
+1. Get the current camera from the scene with `SceneView.getCurrentViewpointCamera()`.
+2. Create a `LocationViewshed`, passing in the `Camera` and a min/max distance.
+3. Update the viewshed from a camera with `viewshed.updateFromCamera(camera)`.
+
+## Relevant API
 
-# Relevant API
 * AnalysisOverlay
-* ArcGISTiledElevationSource
 * ArcGISScene
```
Comment (sorenoid) [inline java/viewshed-camera/README.md]: I wonder why it is necessary to list ArcGISScene in the relevant API? I see that in the relatively more complicated Viewshed GeoElement ArcGISScene doesn't merit a mention, but also in the preceding Point Cloud sample it is glossed over as well, with only the PointCloudLayer listed for API.

File: java/view-content-beneath-the-terrain-surface/README.md [hunk 2]
```diff
@@ -2,36 +2,31 @@
...
 
-2. To allow the camera to go underground, set the surface's navigation
-   constraint to `NONE`.
+* Surface
+* NavigationConstraint
 
-## Relevant API
+## About the data
+
+This data is a point scene layer showing underground wellbore paths (green polylines) and seismic events (brown points).
```
[the remark this answers is not in the document]
Comment (Rachael-E) [reply java/view-content-beneath-the-terrain-surface/README.md]: I guess they're a browny-orange/orangey brown?! I'll go with light brown.

File: java/viewshed-camera/README.md (hunk 1 above)
Comment (Rachael-E) [reply java/viewshed-camera/README.md]: you're right, that should come out along with SceneView. They aren't the main act of this sample. I've removed them, thanks!

**Comment by Rachael-E** [comment]
thanks again @sorenoid! again because these changes are small I'll go ahead and merge.

**Pull request merged**